contexts/credentials/v2

```diff
@@ -60,6 +60,10 @@
         "termsOfUse": {
           "@id": "https://www.w3.org/2018/credentials#termsOfUse",
           "@type": "@id"
+        },
+        "confidenceMethod": {
+          "@id": "https://www.w3.org/2018/credentials#confidenceMethod",
+          "@type": "@id"
```

Comment:
```suggestion
        "confidenceMethod": {
          "@id": "https://www.w3.org/2018/credentials#confidenceMethod",
          "@type": "@id"
```

Shouldn't this be scoped to credentialSubject?

Reply:
It should either be globally scoped or use `"@propagate": true`, but, yes, making it a property on a VC only isn't correct.

Reply:
I believe `confidenceMethod` could be scoped to any property/ies. I don't understand what it would do for `credentialSubject`, nor what it does for `VerifiableCredential`.

Reply:
Based on the comments above, I don't think it should be scoped... However, if it should be scoped, there should be normative statements that people can read, before the context scoping is applied... and some human readable summary of why, here:

https://www.w3.org/2018/credentials#confidenceMethod

Reply:
Maybe https://w3id.org/security#confidenceMethod would be a better term definition, if the W3C CCG will define the behavior.

Reply:
By "scoped to any property/ies" above, I meant that this `confidenceMethod` should be applicable to any property+value pair (or group of same), something like 
```
_:property
   [ _:value          12345 ; 
     confidenceMethod `{uri to method used to get or usable to test value}` 
   ]
```